<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAlert</class>
 <widget class="QDialog" name="dlgAlert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblAlertType">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="sbLen1">
        <property name="maximum">
         <number>5000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblAlertMsg">
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblAlertVal">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblTimeFrame">
        <property name="text">
         <string>Time Frame</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblAlertCond">
        <property name="text">
         <string>Condition</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="dsbAlertVal">
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="cmbAlertType">
        <item>
         <property name="text">
          <string>Price</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Price + EMA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Price + HMA</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="cmbAlertCond">
        <item>
         <property name="text">
          <string>Greater Than</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Less Than</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblStkName">
        <property name="text">
         <string>Stock Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="leStkName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblLen2">
        <property name="text">
         <string>Length2</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="sbLen2">
        <property name="maximum">
         <number>5000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="2">
       <widget class="QTextEdit" name="txteMsg"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblLen1">
        <property name="text">
         <string>Length1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="cmbTimeFrame">
        <item>
         <property name="text">
          <string>Daily</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weekly</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monthly</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSetAlert">
     <property name="text">
      <string>set Alert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
